lib/FrontendTool/ReferenceDependencies.cpp

```diff
@@ -102,6 +102,17 @@ static std::string mangleTypeAsContext(const NominalTypeDecl *type) {
   return Mangler.mangleTypeAsContextUSR(type);
 }
 
+std::vector<std::string>
+swift::reversePathSortedFilenames(const ArrayRef<std::string> elts) {
```

Comment:
Hm, okay. I would actually expect the existing logic to always produce the same order, even in the incremental case. But sorting is always safer.